EasyPost.nuspec

```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>EasyPost-Official</id>
     <title>EasyPost (Official)</title>
-    <version>2.5.1.3</version>
+    <version>2.5.1.4</version>
```

Comment:
We'd like to move to semantic versioning, is that doable here?

Reply:
Version is now 2.6.0